README.adoc

```diff
@@ -24,39 +24,35 @@ Learn how to use Contexts and Dependency Injection to manage and inject dependen
 service on that host to get these system properties. The system properties are then stored and returned.
 
-You will use scopes to bound objects in this application to their well defined-contexts.
+You will use scopes to bound objects in this application to their well-defined contexts.
 CDI provides a variety of scopes for you to work with and while you will not use all of them in this guide,
-there is one for almost every scenario that you may encounter. Scopes are defined using CDI annotations.
+there is one for almost every scenario that you may encounter. Scopes are defined by using CDI annotations.
 You will also use dependency injection to inject one bean into another to make use of its functionalities.
 This enables you to inject the bean in its specified context without having to instantiate it yourself.
 
-The implementation of the application and its services are provided for you under the `start/src` directory.
-The `system` service can be found
-under `start/src/main/java/io/openliberty/guides/system`, and the `inventory` service can be found
-under `start/src/main/java/io/openliberty/guides/inventory`.
-If you want to learn more about RESTful web services and how to build them, see
-https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service]
-which has details on how to build the `system` service.
+The implementation of the application and its services are provided for you in the `start/src` directory.
+The `system` service can be found in `start/src/main/java/io/openliberty/guides/system`, and the `inventory` service can be found
+in `start/src/main/java/io/openliberty/guides/inventory`. If you want to learn more about RESTful web services and how to build them, see
+https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service]for details about how to build the `system` service.
 The `inventory` service is built in a similar way.
 
 === What is CDI?
 
 Contexts and Dependency Injection (CDI) defines a rich set of complementary services that improve the application structure.
-The most fundamental services provided by CDI are Contexts that bind the lifecycle of stateful components to well-defined contexts,
+The most fundamental services that are provided by CDI are Contexts that bind the lifecycle of stateful components to well-defined contexts,
 and Dependency Injection that is the ability to inject components into an application in a typesafe way.
 With CDI, the container does all the daunting work of instantiating dependencies, and
 controlling exactly when and how these components are instantiated and destroyed.
 
-
 == Getting started
 
-The fastest way to work through this guide is to clone the git repository and use the starting project
+The fastest way to work through this guide is to clone the Git repository and use the starting project
 that is provided in the `start` directory. To do this, run the following commands:
```

Comment:
changed since it is a name/product.

Reply:
Ok. This is from the common adoc, which is fixed too.